lib/Compilation.js

```diff
@@ -451,9 +457,9 @@ class Compilation extends Tapable {
 		recursive,
 		callback
 	) {
-		let _this = this;
-		const start = _this.profile && Date.now();
-		const currentProfile = _this.profile && {};
+		let __this = this;
```

Comment:
Could you remove this statement and replace `_this` by `this`? It should be possible because the code below uses arrow functions